src/main/java/net/sf/jabref/gui/cleanup/FieldFormatterCleanupsPanel.java

```diff
@@ -193,8 +217,7 @@ private JPanel getSelectorPanel() {
                         "pref, 2dlu, pref:grow, 2dlu"));
 
         List<String> fieldNames = InternalBibtexFields.getAllPublicFieldNames();
-        fieldNames.add(BibEntry.KEY_FIELD);
-        fieldNames.add("all");
+        fieldNames.addAll(Arrays.asList(BibEntry.KEY_FIELD, FieldName.ABSTRACT_ALL_FIELD, FieldName.ABSTRACT_ALL_TEXT_FIELDS_FIELD));
```

Comment:
and extract it to a new method `InternalBibtexFields.getAllPublicAndInteralFieldNames()`.
Btw, I would suggest to rename the `ABSTRACT` prefix to `INTERNAL` (when reading "FieldName.abstract_Something" I always think of the "abstract" field)